### DISCUSSION TITLE
rabbitmqctl without installing rabbitmq-server or using docker image

### DISCUSSION THREAD

**Discussion by GreatBahram** [rabbitmq/rabbitmq-server#2815] [Other]
Hello there,

Thank you for such an amazing application. I want to control the rabbitmq server just like the `rabbitmqctl` command; more specifically, I want to be able to add, remove users, and assign a specific queue to each user from **outside** of the rabbitmq container. How could I achieve this _without installing_ Rabbitmq-server on my host or using the management interface?

I checked the Rabbitmq HTTP API, and it seems I can declare queues, but it does not support adding or removing users.

Thank you

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
* [Definition import](https://www.rabbitmq.com/definitions.html)
* Download the [Generic binary build](https://www.rabbitmq.com/install-generic-unix.html) and use CLI tools from it (the rest of the installation provides libraries and dependencies)
